src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -186,7 +186,6 @@ public static IEnumerable<object[]> ContentEncoding_TestData()
             });
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
```

Comment:
I might be blind but aren't you removing a conditionaltheory without adding a theory? This test will never run